test/attr/attr_implements_fp.swift

```diff
@@ -1,6 +1,6 @@
 // RUN: %empty-directory(%t)
 // RUN: echo 'main()' >%t/main.swift
-// RUN: %target-swiftc_driver -o %t/a.out %s %t/main.swift -Xfrontend -enable-operator-designated-types -Xfrontend -solver-enable-operator-designated-types
+// RUN: %target-swiftc_driver -Xfrontend -enable-operator-designated-types -Xfrontend -solver-enable-operator-designated-types -o %t/a.out %s %t/main.swift
```

Comment:
If you're improving this test anyway, please use `%target-build-swift` instead of `%target-swiftc_driver`, which includes things like `-module-cache-path`.

Reply:
(Okay, that one specifically is in both, but there are some other options that are only in `target-build-swift`. And I should probably look into whether we still need both.)